Core/src/Utilities/AnnealingUtility.cpp

```diff
@@ -8,14 +8,15 @@
 
 #include "Acts/Utilities/AnnealingUtility.hpp"
 
-/// @brief Gaussian function for weight calculation
+/// @brief Gaussian-like function for weight calculation
```

Comment:
The docs do not match the function name anymore.

Reply:
Why? It's a Gaussian-like function (in fact, it's a real Gaussian function except for a normalisation factor). So I think the docs is pretty accurate here, calling it a "Gaussian-like function for weight calculation".

Reply:
Sorry, I did see that one of the input arguments is already squared. We leave it as-is.